Sources/Core/ObjectiveCInitExtension.swift

```diff
@@ -316,9 +316,9 @@ extension ObjCModelRenderer {
         return ObjCIR.method("- (instancetype)initWithModelDictionary:(NS_VALID_UNTIL_END_OF_SCOPE NSDictionary *)modelDictionary error:(NSError *__autoreleasing *)error") {
             [
                 "NSParameterAssert(modelDictionary);",
-                ObjCIR.ifStmt("!modelDictionary") { ["return self;"] },
```

Comment:
alternatively, can we just return nil if the dictionary is nil ( and can't deserialized into a model object)?

Reply:
after this change, a nil dictionary is like an empty dictionary. The "model" is fully instantiated.

Typically -init methods will not intentionally return `nil`. Typically class create methods will return `nil`.

Reply:
makes sense and you don't change semantic here.

(I was thinking about nil model vs an empty model when server returns bad response, but that's different issue.)